src/modules/rules/tool-clean-extra.ts

```diff
@@ -0,0 +1,74 @@
+import { useDialog } from "../../utils/dialog";
+import { defineRule } from "./rule-base";
+
+interface Options {
+  fieldsToClean?: string[];
+}
+
+export const ToolCleanExtra = defineRule<Options>({
+  id: "tool-clean-extra",
+  scope: "field",
+  targetItemTypes: ["journalArticle"],
+  targetItemField: "title",
+  async apply({ item, options, debug }) {
+    const extras = ztoolkit.ExtraField.getExtraFields(item);
+
+    for (const fieldToClean of options.fieldsToClean ?? []) {
+      if (extras.has(fieldToClean)) {
+        extras.delete(fieldToClean);
+      }
+    }
+    debug("Cleaned extra fields:", extras);
+
+    await ztoolkit.ExtraField.replaceExtraFields(item, extras);
+  },
+
+  async getOptions({ items }) {
+    const fields = new Set<string>();
+
+    for (const item of items) {
+      const extras = ztoolkit.ExtraField.getExtraFields(item);
+      for (const [field, _value] of extras) {
+        fields.add(field);
+      }
+    }
+
+    const fieldPrefix = "field--" as const;
+    const { dialog, open } = useDialog<{
+      [key: `${typeof fieldPrefix}${string}`]: boolean;
+    }>();
+
+    dialog
+      .addStaticRow("Select Fields to Clean", {
+        tag: "label",
+      });
+
+    fields.forEach((extra) => {
+      dialog.addSetting(`${extra}`, `${fieldPrefix}${CSS.escape(extra)}`, {
```

Comment:
```suggestion
      dialog.addSetting(`${extra}`, `${fieldPrefix}${extra}`, {
```

needs: https://github.com/windingwind/zotero-plugin-toolkit/pull/93